terraform.tfvars.example

```diff
@@ -179,7 +179,7 @@ image_signing_keys = []
 check_node_active = "all"
 enable_pv_encryption_in_transit = false
 node_pools = {
-  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, label = { app = "frontend", pool = "np1" } }
+  np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, label = { app = "frontend", pool = "np1" }, freeform_tags = { app = "frontend", pool = "np1" }, defined_tags = { "Operations.CostCenter" = "42" } }
```

Comment:
This defined _tags for specific nodepools worker nodes are not coming through. The one specified for all worker nodes below are coming through.

Reply:
Can you please check again?

Reply:
This example sets tags specific to nodepool `np1` but it doesn't apply them to the single nodes because the keys to define them are `freeform_node_tags` and `defined_node_tags`, as you can see from `modules/oke/nodepools.tf:56`

If you want I can add a coalesce function to perform the merge between the specific mapping for the nodes or the one of the node pool, if the first one is not defined. This should ensure the maximum flexibility but let me know how you think it could be solved.

Reply:
I see the problem now. It's actually working which is great. However, there is potential for confusion.

For node pools, you are looking up `freeform_tags` and `defined_tags` whereas for worker nodes, you are looking up  `freeform_node_tags` and `defined_node_tags`.

Can you please standardize the tags as follows:

For node pools:
- nodepool_freeform_tags
- nodepool_defined_tags

and worker nodes:
- node_freeform_tags
- node_defined_tags

Also, please update the example in `terraform.tfvars.example`?

One more round and we should be able to merge!

Thanks again for your patience.

Reply:
Hi @hyder!
I just updated it, so it should be fine now.